libs/krew-angular/auth/src/lib/login.component.html

```diff
@@ -1,87 +1,5 @@
-          <form action="#" method="POST" class="space-y-6">
-            <div>
-              <div class="mt-1">
-                <input
-                  id="email"
-                  name="email"
-                  type="email"
-                  autocomplete="email"
-                  required
-                  placeholder="Email address"
-                  class="block w-full px-3 py-2 text-white rounded-md shadow-sm appearance-none  bg-brand-01-darker placeholder-brand-01-placeholder focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
-                />
-              </div>
-            </div>
-
-            <div class="space-y-1">
-              <label for="password" class="block text-sm font-medium text-gray-700"> Password </label>
-              <div class="mt-1">
-                <input
-                  id="password"
-                  name="password"
-                  type="password"
-                  autocomplete="current-password"
-                  required
-                  class="block w-full px-3 py-2 placeholder-gray-400 border border-gray-300 rounded-md shadow-sm appearance-none  focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm"
-                />
-              </div>
-            </div>
-
-            <div class="flex items-center justify-between">
-              <div class="flex items-center">
-                <input
-                  id="remember-me"
-                  name="remember-me"
-                  type="checkbox"
-                  class="w-4 h-4 text-indigo-600 border-gray-300 rounded focus:ring-indigo-500"
-                />
-                <label for="remember-me" class="block ml-2 text-sm text-gray-900"> Remember me </label>
-              </div>
-
-              <div class="text-sm">
-                <a href="#" class="font-medium text-indigo-600 hover:text-indigo-500"> Forgot your password? </a>
-              </div>
-            </div>
-
-            <div>
-              <button
-                type="submit"
-                class="flex justify-center w-full px-4 py-2 text-sm font-medium text-white bg-indigo-600 border border-transparent rounded-md shadow-sm  hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500"
-              >
-                Sign in
-              </button>
-            </div>
-          </form>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="relative flex-1 hidden w-0 lg:block">
-    <img
-      class="absolute inset-0 object-cover w-full h-full"
-      src="https://images.unsplash.com/photo-1505904267569-f02eaeb45a4c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1908&q=80"
-      alt=""
-    />
-  </div>
-</div>
+<div class="w-screen h-[10rem] bg-red-500/[0.12]"></div>
+<div class="w-screen h-[10rem] bg-green-500/[0.12]"></div>
+<div class="w-screen h-[10rem] bg-brand-01"></div>
+<div class="w-screen h-[10rem] bg-brand-01/50"></div>
```

Comment:
With the tailwind shorthand `/50` here, there is an opacity applied to the theme color brand-01